src/main/java/com/endava/Ticket_Management_System/model/Customer.java

```diff
@@ -0,0 +1,55 @@
+package com.endava.Ticket_Management_System.model;
+
+import jakarta.persistence.*;
+
+import java.io.Serializable;
+
+@Entity
+@Table(name = "Customer")
+public class Customer implements Serializable {
+
+    @Id
+    @Column(name = "customerID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int customer_id;
+
+    @Column(name = "customerName", length = 70)
+    private String customer_name;
+
+    @Column(name = "email", length = 70)
+    private String email;
+
+    public Customer() {
```

Comment:
Why didn't you use lombok for constructors, getter and setters? I see you have it as a dependency